.github/workflows/test.yml

```diff
@@ -0,0 +1,41 @@
+# This is a workflow for ensuring tests pass with the configured git submodules.
+# It does not attempt to update the repos/branches specified in any submodules.
+---
+name: Test
+
+on:
+  # Allow running this workflow manually from the Actions tab
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
```

Comment:
The `docs` workflow runs on `main`. Does this workflow provide any extra benefit there?

```suggestion
```